data/autoyast_sle12/create_hdd/create_hdd_ha_sles.xml.ep

```diff
@@ -305,25 +305,66 @@
         <products config:type="list">
             <product><%= uc $get_var->('SLE_PRODUCT') %></product>
         </products>
-        <!-- Workaround for bsc#1202234: [addon]-release packages are missing after autoyast installation. -->
         <packages config:type="list">
-        % foreach (values %$addons) {
+            <!-- Workaround for bsc#1202234: [addon]-release packages are missing after autoyast installation. -->
```

Comment:
not sure yet if this would work for sle 15 ha, we could just delete the sp from the name if works for both sle 12.

Reply:
That profile is in autoyast_sle12 it's not for sle 15, effectively it seems to be missing mapping in registration.pm for sle 15, but we can make adjustements in the future if needed